### ISSUE TITLE
Error using live search

### ISSUE THREAD

**Issue by Six-wars** [snapappointments/bootstrap-select#1637]
I'm getting the following error when i type into the search

```
Uncaught TypeError: Cannot read property '3' of undefined
    at $.expr.pseudos.icontains (bootstrap-select.js:199)
    at jquery.js:15
    at jquery.js:15
    at jquery.js:15
    at bm (jquery.js:15)
    at bm (jquery.js:15)
    at Z (jquery.js:15)
    at Function.Z.matches (jquery.js:15)
    at Function.filter (jquery.js:15)
    at bj (jquery.js:15)
```

[full details](http://stackoverflow.com/questions/42206170/boostrap-select-search-not-working) of error.

**Comment by caseyjhol** [comment]
Can you link me to a live example?

**Comment by Six-wars** [comment]
Here's the link http://kazichimp.com/accounts/login/?next=/register/profile_home

Username: test@example.com
password: test

**Comment by caseyjhol** [comment]
Fixed in v1.13.0.
